data/utils.py

```diff
@@ -2,6 +2,47 @@
 from pathlib import Path
 
 from PIL import Image
+from ruamel.yaml import YAML
+
+yaml = YAML(typ="rt")
+
+TRANSLATION_BUFFER_PATH = Path(__file__).parent / "translations.yaml"
+
+with open(TRANSLATION_BUFFER_PATH, encoding="utf-8") as yaml_file:
+    TRANSLATION_BUFFER = yaml.load(yaml_file)
+
+
+class TranslatableStr(dict):
+    """
+    Wrapper for translations.
+    Takes a string, that should be translated and looks it up in the translation buffer.
+    If the string is not found, it an entry is created in the buffer.
+
+    The string can be formatted using the .format() method or left as is.
+
+    since subclassing from dict, this class does not need any aditional infrasturcture
+    to turn a message into a translated string.
```

Comment:
```suggestion
    Since it is a subclass of dict, this class does not need any additional infrastructure
    to turn a message into a translated string.
    
    Translatable strings will be exported as {"de": "<de string>", "en": "<en string>"}.
```